src/winetricks

```diff
@@ -11069,6 +11069,51 @@ load_xact()
 
 #----------------------------------------------------------------
 
+if test "$W_ARCH" = "win64"; then
```

Comment:
This is unnecessary (and will probably cause problems in GUI mode)

Reply:
This was so the `$W_SYSTEM64_DLLS_WIN64` below would expand correctly. But removed